src/client/pages/main/components/Pics/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
```

Comment:
Please use kebabCase for names of the directories
`Pics` -> `pics`
`Video` -> `video`